### ISSUE TITLE
Allow sending customStyleMap to convertFromHTML

### ISSUE THREAD

**Issue by adamtal3** [facebook/draft-js#499]
I tried digging into `convertFromHTML` and found no way to pass something which will read inline styles.

The function I found that parses inline styles doesn't receive any outside mapping and maps only: `bold`, `italic`, `underline` and `strikethrough`.

My wish is to use the [Colorful Editor](http://github.com/facebook/draft-js/tree/master/examples/color) example together with `convertFromHTML`.

Is there something I'm missing?
Can the `DOMBuilder` or `blockRenderMap` arguments help me with that?

**Comment by hellendag** [comment]
Yeah, this is pretty restrictive right now. I think it would make sense to have better control over inline style handling for pastes. @clowbrow, any chance you've spent time thinking about this?
